pyodide_build/tests/test_venv.py

```diff
@@ -256,3 +258,36 @@ def test_pip_downgrade(base_test_dir):
     assert result.returncode == 0, result.stderr
     assert result.stdout.startswith("pip 24.0")
     assert venv_pip_path.readlink() == venv_pip_path.with_name("pip_patched")
+
+
+def test_pytest_invoke(base_test_dir):
```

Comment:
```suggestion
@pytest.mark.integration
def test_pytest_invoke(base_test_dir):
```

I think this test needs an integration marker.